### PR TITLE
Added new journey for FI user

### PR DESCRIPTION
**Pull request by awarhade**

### CODE REVIEW

File: src/test/scala/uk/gov/hmrc/test/ui/pages/FiIsTheAddressCorrectPage.scala [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ */
+
+package uk.gov.hmrc.test.ui.pages
+
+import org.openqa.selenium.By
+
+object FiIsTheAddressCorrectPage extends BasePage {
+
+  override val pageUrl: String = baseUrl + "/registered-business/is-the-address-correct"
+  val haveUtrYesId: By         = By.id("value")
```
Comment (C-Thomp) [inline src/test/scala/uk/gov/hmrc/test/ui/pages/FiIsTheAddressCorrectPage.scala:24]: haveUtrYesId -> should this be something like 'addressCorrectYes'?

File: src/test/scala/uk/gov/hmrc/test/ui/pages/FiIsThisYourBusinessNamePage.scala [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ */
+
+package uk.gov.hmrc.test.ui.pages
+
+import org.openqa.selenium.By
+
+object FiIsThisYourBusinessNamePage extends BasePage {
+
+  override val pageUrl: String = baseUrl + "/registered-business/is-this-your-business-name"
+  val haveUtrYesId: By         = By.id("value")
```
[the remark this answers is not in the document]
Comment (C-Thomp) [reply src/test/scala/uk/gov/hmrc/test/ui/pages/FiIsThisYourBusinessNamePage.scala:24]: infact if they are always the same for everypage put them into the BasePage file :)

[the remark this answers is not in the document]
Comment (awarhade) [reply src/test/scala/uk/gov/hmrc/test/ui/pages/FiIsThisYourBusinessNamePage.scala:24]: true, I will add it in basepage file

**Review by C-Thomp — APPROVED** [review]
🥳

**Pull request merged**